app/scripts/controllers/auth.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name todoApp.controller:AboutCtrl
+ * @description
+ * # AboutCtrl
+ * Controller of the todoApp
+ */
+angular.module('todoApp')
+  .controller('authCtrl', function ($scope, $window, User, ngNotify) {
+
+    //Initializing our user object
+    $scope.user = {};
+
+    //Initializing our user token
+    $scope.currentSavedToken = localStorage.getItem("token");
+
+    //registering the user
+    $scope.register = function() {
+      User.register($scope.user, function(response) {
+        ngNotify.set('You are now Registered. Redirecting...', 'success');
+        console.log(response);
```

Comment:
Remove these console.log() statements. Simple find and replace (with regex) should do it.